cli/cmd/integration_docker_hub.go

```diff
@@ -72,13 +77,14 @@ func createDockerHubIntegration() error {
 	}
 
 	answers := struct {
-		Name           string
-		Username       string
-		Password       string
-		LimitTag       string `survey:"limit_tag"`
-		LimitLabel     string `survey:"limit_label"`
-		LimitRepos     string `survey:"limit_repos"`
-		LimitMaxImages string `survey:"limit_max_images"`
+		Name                string
+		Username            string
+		Password            string
+		LimitTag            string `survey:"limit_tag"`
+		LimitLabel          string `survey:"limit_label"`
+		LimitRepos          string `survey:"limit_repos"`
+		LimitMaxImages      string `survey:"limit_max_images"`
+		NonOSPackageSupport bool   `survey:"non_os_package_support"`
```

Comment:
Nit: Let's pick a consolidated name for the field. Do we want it to be called `NonOSPackageSupport` or `NonOSPackageEval`. I personally think support is a little clearer than eval.